### ISSUE TITLE
Automate template_version bumps used to version docker bosh cli / ops portal

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/paas-templates#440]
### Expected behavior

As a template author, in order to ensure docker bosh cli / ops portal properly deploy bumps, I need automated management of paas-template version used in v41 docker bosh cli / ops portal automation

### Observed behavior

paas-template/micro-depls/credhub-seeder/template/meta-template-version-operators.yml has hard coded version that requires manual bumps to trigger docker bosh cli / ops portal deployments

```
 #--- Load template version properties to credhub
- type: replace
  path: /properties/credhub/import_credentials/-
  value:
    name: /meta/template_version
    type: value
    value: "40.0.0-RC6" 
```

### Affected release

Reproduced on version 41.RC1

**Comment by poblin-orange** [comment]
might use some git templating to insert release value @tag time.

**Comment by gberche-orange** [comment]
closed as duplicate of #415
